test/packaging/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+#
+# CMakeLists.txt
+#
+#
+# The MIT License
+#
+# Copyright (c) 2022 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# Expects invocation wtih -DCMAKE_PREFIX_PATH=<directory-of-tiledb-installation>
+
+cmake_minimum_required(VERSION 3.21)
+
+project(Test_TileDBConfigVersion_h NONE)
+
+if (NOT CMAKE_PREFIX_PATH)
+  message(FATAL_ERROR "Need parameter -DCMAKE_PREFIX_PATH=<tiledb-installation-path>")
+else()
+  if (NOT EXISTS "${CMAKE_PREFIX_PATH}/include/TileDB/tiledb_version.h")
+    message(FATAL_ERROR "Missing tiledb_version.h at ${CMAKE_PREFIX_PATH}")
+  endif()
+endif()
+
+# Parse version numbers out of tiledb_version.h file
+
+file(STRINGS ${CMAKE_PREFIX_PATH}/include/TileDB/tiledb_version.h
+  version_h_lines
+)
+
+
+set(SEARCH_VARIABLES "TILEDB_VERSION_MAJOR" "TILEDB_VERSION_MINOR" "TILEDB_VERSION_PATCH")

```

Comment:
Any reason not to use https://github.com/TileDB-Inc/TileDB/blob/1c54984db4a5bdca6ae4012b96ccf395a1ba33f9/tiledb/CMakeLists.txt#L1057-L1068  ?